config/deploy/stage.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 server 'preservation-catalog-stage-01.stanford.edu', user: 'pres', roles: %w[app db web]
-server 'preservation-catalog-stage-02.stanford.edu', user: 'pres', roles: %w[resque app]
+server 'preservation-catalog-stage-02.stanford.edu', user: 'pres', roles: %w[app resque queue_populator cache_cleaner]
```

Comment:
`queue_populator` is what queues up the various weekly audit jobs.  i don't think anyone had noticed this not running on stage, i only noticed it when fixing the `cache_cleaner` issue.